cross-country/openrally.xsd

```diff
@@ -0,0 +1,145 @@
+  <xs:element name="fz" type="markerType"/>
+  <!-- Duplicate of "fz" <xs:element name="endofzone" type="markerType"/> -->
+  <!-- WP[x] waypoints -->
+  <xs:element name="wpm" type="waypointType"/>
+  <xs:element name="wpe" type="waypointType"/>
+  <xs:element name="wps" type="waypointType"/>
+  <xs:element name="wpc" type="waypointType"/>
+  <!--
+	Duplicate of "wpe" <xs:element name="waypointeclipse" type="markerType"/>
+	Duplicate of "wps" <xs:element name="waypointsafety" type="markerType"/>
+	Duplicate of "wpm" <xs:element name="waypointmasked" type="markerType"/>
+	-->
+  <!-- drawing elements -->
+  <xs:element name="tulip" type="drawingType"/>
+  <xs:element name="notes" type="drawingType"/>
+  <!-- other control points -->
+  <xs:element name="stop" type="timeType"/>
+  <xs:element name="checkpoint" type="markerType"/>
+  <xs:element name="neutralization" type="timeType"/>
+  <xs:element name="timecontrol" type="markerType"/>
+  <xs:element name="fuel" type="markerType"/>
+  <xs:element name="reset" type="distanceType"/>
+  <!--
+	Duplicate of "stop" <xs:element name="stopcontrol" type="markerType"/>
+	Replaced by reset <xs:element name="resettozero" type="markerType"/>
+  -->
+
+  <!--
+  safety tags, primarily used in an org's mapping/tracking to locate
+  assets on the course.
+  -->
+  <xs:element name="medical" type="markerType"/>
+	<xs:element name="helicopter" type="markerType"/>
+	<xs:element name="police" type="markerType"/>
+	<xs:element name="fire" type="markerType"/>
+	<xs:element name="spectators" type="markerType"/>
+  <xs:element name="media" type="markerType"/>
+
+  <!--
+  Propose removal from cross-country. Unknown practical use here. Some of these
+  were from road rally users and will remain in their XSD
+	<xs:element name="gps" type="markerType"/>
+	<xs:element name="passagecontrol" type="markerType"/>
+	<xs:element name="endfuel" type="markerType"/>
+	<xs:element name="radio" type="markerType"/>
+	<xs:element name="begintireservice" type="markerType"/>
+	<xs:element name="endtireservice" type="markerType"/>
+	<xs:element name="toilets" type="markerType"/>
+	<xs:element name="giveway" type="markerType"/>
+	<xs:element name="sensitivezone" type="markerType"/>
+	<xs:element name="startofassistance" type="markerType"/>
+	<xs:element name="finishofassistance" type="markerType"/>
+	<xs:element name="startofzone" type="markerType"/>
+	<xs:element name="village" type="markerType"/>
+  -->
+
+  <!-- data types -->
+  <xs:complexType name="waypointType">
+    <xs:simpleContent>
+      <xs:extension base="xs:string">
+        <xs:attribute type="xs:string" name="name"/>
+        <xs:attribute type="xs:positiveInteger" name="open"/>
+        <xs:attribute type="xs:positiveInteger" name="clear"/>
+      </xs:extension>
+    </xs:simpleContent>
+  </xs:complexType>
+  <xs:simpleType name="unitsType">
+    <xs:restriction base="xs:NMTOKEN">
+      <xs:enumeration value="metric"/>
+      <xs:enumeration value="statue"/>
```

Comment:
"statute"?

Reply:
Sorry. I never saw these comments... https://en.wikipedia.org/wiki/Mile#Statute_mile

Reply:
No worries - I was just pointing out what appears to be a spelling mistake: "statue" should be "statute", no?